### ISSUE TITLE
 PFE-pytorch

### ISSUE THREAD

**Issue by Ontheway361** [seasonSH/Probabilistic-Face-Embeddings#12]
this is my PFE-Pytorch-version[https://github.com/Ontheway361/pfe-pytorch], but it is something amazing that there is neg mls-loss during the training...

**Comment by HeshamAMH** [comment]
please  I need a link for the pretrained model and the result on LFW.
